lg_mirror/webapps/playback/index.html

```diff
@@ -28,18 +30,40 @@
 
 var streamDiv = document.getElementById('stream');
 
+var rosbridgeHost = getParameterByName('rosbridge_host', String, 'localhost');
+var rosbridgePort = getParameterByName('rosbridge_port', Number, 9090);
+var rosbridgeSecure = getParameterByName('rosbridge_secure', stringToBoolean, false);
+var rosbridgeUrl = getRosbridgeUrl(rosbridgeHost, rosbridgePort, rosbridgeSecure);
+var rosInstanceName = getParameterByName('ros_instance_name', String);
 var webVideoServerUri = getParameterByName('web_video_server', String, 'http://localhost:8080');
 var viewport = getParameterByName('played_back_viewport', String);
 if (!viewport) {
   console.error('Must specify viewport param');
 }
 
+var ros = new ROSLIB.Ros();
```

Comment:
this whole block is done in [this](https://github.com/EndPointCorp/lg_ros_nodes/tree/master/lg_common/src/lg_common/extensions/ros_window_ready) extension - it gets loaded whenever you set `preload` to `true`